fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -4540,8 +4407,10 @@ class FMCMainDisplay extends BaseAirliners {
      * @param {string} s place-bearing/place-bearing
      * @throws {McduMessage}
      * @returns {[WayPoint, number, WayPoint, number]} place and true bearing * 2
+     * @deprecated use {@link Fmgc.WaypointEntryUtils.parsePbx}
      */
     async parsePbx(s) {
+        return Fmgc.WaypointEntryUtils.parsePbx(this, s);
         const pbx = s.match(/^([^\-\/]+)\-([0-9]{1,3})\/([^\-\/]+)\-([0-9]{1,3})$/);
         if (pbx === null) {
             throw NXSystemMessages.formatError;
```

Comment:
```suggestion
```
Ditto, dead code, check that this 
```

      const pbx = s.match(/^([^\-\/]+)\-([0-9]{1,3})\/([^\-\/]+)\-([0-9]{1,3})$/);
      if (pbx === null) {
          throw NXSystemMessages.formatError;
```
regex matching is included in parsePbx WayPointEntryUtils